react/components/molecules/key-value/key-value.js

```diff
@@ -18,7 +18,10 @@ export class KeyValue extends PureComponent {
             iconHeight: PropTypes.number,
             iconWidth: PropTypes.number,
             iconStrokeWidth: PropTypes.number,
-            iconOnPress: PropTypes.func
+            clickable: PropTypes.boolean,

```

Comment:
In mobile there are presses not clicks :D, please rename

Reply:
Creating a PR to rename this as well on `ripe-robin-revamp`